accounting/rewind.go

```diff
@@ -145,7 +145,8 @@ func AccountAtRound(account models.Account, round uint64, db idb.IndexerDb) (acc
 			}
 		case atypes.AssetFreezeTx:
 		default:
-			panic(fmt.Sprintf("unknown txn type %s", stxn.Txn.Type))
+			err = fmt.Errorf("%s[%d,%d]: cannot rewind past txn type %s", account.Address, txnrow.Round, txnrow.Intra, stxn.Txn.Type)
```

Comment:
It's not totally clear that the reason why it cannot rewind is that it isn't supported. Maybe something like `txn type %s is not currently supported by rewind`?

Reply:
Updated with variation on this language